x/emissions/module/rewards/rewards_test.go

```diff
@@ -1061,6 +1071,165 @@ func (s *RewardsTestSuite) TestIncreasingTaskRewardAlphaIncreasesImportanceOfPre
+	require.NoError(err)
+	require.True(notFound)
+
+	worker1_0, notFound, err := k.GetInfererNetworkRegret(s.ctx, topicId0, sdk.AccAddress(workerAddrs[1].String()))
+	require.NoError(err)
+	require.True(notFound)
+
+	worker2_0, notFound, err := k.GetInfererNetworkRegret(s.ctx, topicId0, sdk.AccAddress(workerAddrs[2].String()))
+	require.NoError(err)
+	require.True(notFound)
+
+	/// TEST 0 PART A
+
+	s.getRewardsDistribution(
+		topicId0,
+		blockHeight0,
+		workerValues,
+		reputerValues,
+		workerAddrs[0],
+		"0.1",
+		"0.1",
+	)
+
+	/// TEST 0 PART B
+
+	blockHeight1 := blockHeight0 + blockHeightDelta
+	s.ctx = s.ctx.WithBlockHeight(blockHeight1)
+
+	s.getRewardsDistribution(
+		topicId0,
+		blockHeight1,
+		workerValues,
+		reputerValues,
+		workerAddrs[0],
+		"0.1",
+		"0.2",
+	)
+
+	worker0_0, notFound, err = k.GetInfererNetworkRegret(s.ctx, topicId0, sdk.AccAddress(workerAddrs[0].String()))
+	require.NoError(err)
+	require.False(notFound)
+
+	worker1_0, notFound, err = k.GetInfererNetworkRegret(s.ctx, topicId0, sdk.AccAddress(workerAddrs[1].String()))
+	require.NoError(err)
+	require.False(notFound)
+
+	worker2_0, notFound, err = k.GetInfererNetworkRegret(s.ctx, topicId0, sdk.AccAddress(workerAddrs[2].String()))
+	require.NoError(err)
+	require.False(notFound)
+
+	/// INCREASE ALPHA REGRET
+
+	currentParams.AlphaRegret = alloraMath.MustNewDecFromString(("0.2"))
+	err = k.SetParams(s.ctx, currentParams)
+	require.NoError(err)
+
+	/// TEST 1 PART A
+
+	blockHeight2 := blockHeight1 + blockHeightDelta
+	s.ctx = s.ctx.WithBlockHeight(blockHeight2)
+
+	topicId1 := s.setUpTopic(blockHeight2, workerAddrs, reputerAddrs, stake)
+
+	s.getRewardsDistribution(
+		topicId1,
+		blockHeight2,
+		workerValues,
+		reputerValues,
+		workerAddrs[0],
+		"0.1",
+		"0.1",
+	)
+
+	/// TEST 1 PART B
+
+	blockHeight3 := blockHeight2 + blockHeightDelta
+	s.ctx = s.ctx.WithBlockHeight(blockHeight3)
+
+	s.getRewardsDistribution(
+		topicId1,
+		blockHeight3,
+		workerValues,
+		reputerValues,
+		workerAddrs[0],
+		"0.1",
+		"0.2",
+	)
+
+	blockHeight4 := blockHeight3 + blockHeightDelta
+	s.ctx = s.ctx.WithBlockHeight(blockHeight4)
+
+	worker0_1, notFound, err := k.GetInfererNetworkRegret(s.ctx, topicId1, sdk.AccAddress(workerAddrs[0].String()))
+	require.NoError(err)
+	require.False(notFound)
+
+	worker1_1, notFound, err := k.GetInfererNetworkRegret(s.ctx, topicId1, sdk.AccAddress(workerAddrs[1].String()))
+	require.NoError(err)
+	require.False(notFound)
+
+	worker2_1, notFound, err := k.GetInfererNetworkRegret(s.ctx, topicId1, sdk.AccAddress(workerAddrs[2].String()))
+	require.NoError(err)
+	require.False(notFound)
+
+	require.True(worker0_0.Value.Gt(worker0_1.Value))
+	require.True(alloraMath.InDelta(worker1_0.Value, worker1_1.Value, alloraMath.MustNewDecFromString("0.00001")))
+	require.True(alloraMath.InDelta(worker2_0.Value, worker2_1.Value, alloraMath.MustNewDecFromString("0.00001")))
```

Comment:
> We have 2 trials with 2 epochs each, and the first worker does worse in 2nd epoch in both trials, enacted by their increasing loss between epochs.
> We increase alpha between the trials to prove that their worsening performance exaggerates their regret.

that's the intention, right?

^that as a comment would improve this test's clarity tremendously.
your tests in general could benefit from quick snippets of clarity like that.

but also, is this test showing the opposite is true: cuz `worker0_0 > worker0_1`
=> in the second trial, we ultimately regret the worker less, though the weight of their worsening performance was increased
How do we explain that?

Reply:
nvm I'm wrong. regret is more like “how grateful we are for the worker”/“how much we’d regret not having them”, less “how much we regret to include them”